redash/handlers/data_sources.py

```diff
@@ -199,6 +199,15 @@ def get(self, data_source_id):
         except Exception:
             response["error"] = {"code": 2, "message": "Error retrieving schema."}
 
+        try:
+            sorted_schema = [{"name": i['name'], "columns": sorted(i['columns'])} for i in schema]
+            response["schema"] = sorted_schema
```

Comment:
Shouldn't it sort tables as well (or it already does)?

Reply:
The tables are sorted in API handler. It makes sense to move it here, though.

https://github.com/getredash/redash/blob/19c6d331b6f2d22006bfcda3feab12c86c076afc/redash/models/__init__.py#L191

Reply:
I pushed a commit that moves table name sorting to the schema handler.